data/dbConfig.js

```diff
@@ -2,6 +2,10 @@ const knex = require('knex');
 
 const config = require('../knexfile.js');
 
+<<<<<<< HEAD
+const dbEnv = process.env.DATABASE_ENV || 'development';
+=======
 const dbEnv = process.env.DB_ENV || 'development';
+>>>>>>> 983bac2b85b1a6fbe824e3f58083c23df07749a5
```

Comment:
I'm seeing signs of conflicts here, can you please resolve it.

Reply:
Fixed.. this was some error where path was wrong on master